bin/crystal

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
Not sure which is the best, the old one is more portable, as `sh` is not always in `/bin/` on all systems (even if it's unlikely).